iep-lwcapi/src/main/scala/com/netflix/iep/lwcapi/EddaObservable.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2014-2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.iep.lwcapi
+
+import java.io.IOException
+
+import com.fasterxml.jackson.databind.JsonMappingException
+import com.netflix.iep.json.Json
+import com.typesafe.scalalogging.StrictLogging
+import rx.lang.scala.Observable
+
+import scala.util.control.NonFatal
+import scalaj.http._
+
+//
+// Return a set of instances we should try connecting to.
+// As we do not send to these nodes, we only collect data from them,
+// it's important we connect before they are accepting client traffic
+// so we don't drop data.
+//
+object EddaObservable extends StrictLogging {
+  //private val LOGGER = LoggerFactory.getLogger(this.getClass) // TODO: log properly
+
+  private def makeRequest(region: String, environment: String, cluster: String) = {
+    val headers = Map(
+      "Accept" -> "application/json",
+      "X-Netflix.client.cluster.name" -> sys.env.getOrElse("NETFLIX_CLUSTER", "local"),
+      "X-Netflix.environment" -> sys.env.getOrElse("NETFLIX_ENVIRONMENT", "local"),
+      "X-Netflix.client.asg.name" -> sys.env.getOrElse("NETFLIX_AUTO_SCALE_GROUP", "local"),
+      "X-Netflix.client.instid" -> sys.env.getOrElse("EC2_INSTANCE_ID", "local"),
+      "X-Netflix.client.az" -> sys.env.getOrElse("EC2_AVAILABILITY_ZONE", "none"),
+      "X-Netflix.client.region" -> sys.env.getOrElse("EC2_REGION", "none")
+    )
+    Http(s"http://edda-iep.$region.iep$environment.netflix.net:7001/api/v2/group/clusters/$cluster")
+      .headers(headers)
+  }
+
+  case class EddaAutoScalingGroup(name: String,
+                                  start: Long,
+                                  slot: Int)
```

Comment:
Personally I prefer to avoid alignment with the opening paren. I find it harder to read and it causes unnecessary diffs when refactoring that changes where the start is. If it won't fit on the line I prefer:

```scala
case class EddaAutoScalingGroup(
    name: String,
    start: Long,
    slot: Int)
```